generateFakeParticipants.js

```diff
@@ -0,0 +1,49 @@
+require("dotenv").config();
+const faker = require("faker/locale/de");
+const { MongoClient } = require("mongodb");
+
+const dbclient = new MongoClient(process.env.MONGO_URI, {
+  useUnifiedTopology: true,
+});
+
+function getRandomeBirth() {
```

Comment:
```suggestion
function getRandomBirth() {
```
Just a small typo. 
Maybe adding "Date" at the end is an option too.